tox.ini

```diff
@@ -52,9 +52,12 @@ show-source = True
 # E123, E125 skipped as they are invalid PEP-8.
 # N802 skipped (function name should be lowercase)
 # N806 skipped (variable in function should be lowercase)
-ignore = E123,E125,N802,N806
+# F401 skipped (imported but unused) after verifying current usage is valid
+# W503 skipped line break before binary operator
+# C901 skipped: 'MLaunchTool.init' is too complex
+ignore = E123,E125,N802,N806,F401,W503,C901
```

Comment:
An alternative to ignoring C901 is to set `max-complexity = 55` below. Possibly a minor benefit in that it will flag a change that would raise the complexity even higher. Ideally, we would do some refactoring in the future to reduce the complexity of `MLaunchTool.init`, since it is twice as complex as the next most complex function (`MLogFilterTool.run` = 27). It is a low priority though, and it is fine to just ignore that test for now.

Reply:
`MLaunchTool.init` should ideally be refactored in future, but given there weren't any other methods approaching this complexity I didn't think there was benefit in keeping the rule enabled.